scripts/e2e-v6.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -x
+
+source ${OKTA_HOME}/${REPO}/scripts/setup.sh
+
+setup_service xvfb start
+setup_service java 1.8.222
+setup_service google-chrome-stable 87.0.4280.66-1
+
+export TEST_SUITE_TYPE="junit"
+export TEST_RESULT_FILE_DIR="${REPO}/test-reports/e2e"
+
+export ISSUER=https://samples-javascript.okta.com/oauth2/default
+export SPA_CLIENT_ID=0oapmwm72082GXal14x6
+export USERNAME=george@acme.com
+get_secret prod/okta-sdk-vars/password PASSWORD
+
+if ! yarn add -DW --ignore-scripts @okta/okta-auth-js@^6; then
```

Comment:
The [setup.sh script](https://github.com/okta/okta-vue/blob/master/scripts/setup.sh#L6) is designed to work with an environment variable `AUTHJS_VERSION`. This script could be modified to accept the environment variable from the outside. Then the setup could be written
```
AUTHJS_VERSION=6
source ${OKTA_HOME}/${REPO}/scripts/setup.sh
```

We should also avoid duplicating the E2E logic which should be identical between the v6 and v7 tests. I've added a followup task to refactor the scripts in this repo: OKTA-536704